src/imagej/config.py

```diff
@@ -0,0 +1,7 @@
+__author__ = "Curtis Rueden, Michael Pinkert, Ed Evans, Yang Liu, Mark Hiner"
+
+__major__ = 1
+__minor__ = 1
+__patch__ = 2
+__tag__ = "dev0"
+__version__ = f"{__major__}.{__minor__}.{__patch__}.{__tag__}".strip(".")
```

Comment:
This is now duplicated here and in `setup.cfg`.

Is there any way for a module to be able to self-report its version in a DRY manner? I'm concerned these two spots may get out of sync.

Reply:
Probably. I just removed this in 59406c0.

Let's address versioning in a new issue. See https://github.com/imagej/pyimagej/issues/187

Reply:
Drat, I forgot this was called in `__init__.py`

For now, we'll leave the version construction in `setup.py` :confused:

Hopefully we can devise a better way to do the versioning in that issue.